### ISSUE TITLE
Single test run overwrites existing ensemble

### ISSUE THREAD

**Issue by oyvindeide** [equinor/ert#6923]
The single test run overwrites any existing ensemble, which means there is a large change that storage can get out of sync. Should avoid overwriting single realizations in existing ensembles, and in those cases, create a new ensemble instead. Note, this was attempted previously, but had unintended side effects: https://github.com/equinor/ert/issues/6633

**Comment by oyvindeide** [comment]
Blocked by: #7043. I suggest we implement this by adding the option of creating an experiment with a UUID, and have a constant UUID for singel test run, for example UUID=0, that we delete before every single test run. At the same time we should also remove the option of chosing a ensemble in the single test run.

**Comment by oyvindeide** [comment]
Removed the block from this, as we can just create a new experiment and ensemble with ensemble size 1 for now. Once we have deletion I suggest we have a fixed UUID for single test run (UUID=0 for example), and just delete that before each single test run, but that is for another issue.
